app/controllers/admin/slides_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::SlidesController < Admin::ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->